dashboard/app/models/pd/application/application_base.rb

```diff
@@ -110,6 +110,7 @@ class ApplicationBase < ApplicationRecord
     before_create -> {self.status = :unreviewed}
```

Comment:
I *think* we'd want to change the `after_initialize -> {self.status = :unreviewed}, if: :new_record?` line above this one, but I'd like to talk how to do it ... I get errors about status being required, but I can't find where that requirement is being made. Right now, it's a bit funky because the status goes from 'unreviewed' to 'incomplete' immediately upon saving ...